### DISCUSSION TITLE
Manually download metadata option for missing content (Artist images particularly)

### DISCUSSION THREAD

**Discussion by Ay1tsMe** [music-assistant/support#2440] [Q&A] [answered]
I understand that music-assistant used to search in the background for missing metadata and was removed because of performance issues:

![image](https://github.com/music-assistant/hass-music-assistant/assets/110982014/2d80df35-caa6-4519-ba55-fe258f345109)

But is it possible to have an option in the settings to search for metadata anyway. Particularly it would be nice to fetch all the images of artists so that images show up in the artist section of the UI instead of being blank. 

Other metadata is less important as it can music-assistant collects most of the local track data anyway like album covers and so fourth, but it would be nice to atleast have to option to grab it anyway.

**Comment by OzGav** [comment, marked as answer]
No background scanning but data is brought down when you visit the Artist including images. However this is a bug at the moment whereby some artist images are vanishing. Having said all that, note we recommend using local images for efficiency reasons.

edit: this has been suggested as a useful tool to get the images https://github.com/Komet/MediaElch/releases

**Reply by Ay1tsMe** [reply]
Okay so if you can import the artist images locally, how does someone go about doing that? Do you simply drop the image in the same location as the music content and music-assistant will automatically grab it or is there a folder in music-assistant that stores these images?

**Reply by OzGav** [reply]
Follow the instructions here https://music-assistant.io/music-providers/filesystem/
